metrics/baseline/transformers/baseline.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
```

Comment:
Given that we live in the py2 -> py3 transition, a safer hash-bang is:
```python
#!/usr/bin/env python3
```
Especially for the files in ``utils``